### ISSUE TITLE
namespace error in version 2.0

### ISSUE THREAD

**Issue by stevevu** [heyday/silverstripe-versioneddataobjects#37]
Hi,

When clicking on the new 'Versioning ...' button in the CMS, the following error is thrown:

```php
Class 'Heyday\VersionedDataObjects\DataList' not found in /usr/local/www/silverstripe/docroot/silverstripe-versioneddataobjects/code/VersionedDataObjectDetailsForm.php
```

The error occurs on line 251 of VersionedDataObjectDetailsForm.php and cause appears to be that DataList is not namespaced, ie, code should be:

```php
$this->record = \DataList::create($this->record->class)->byID($this->record->ID);
```
 instead of
```
$this->record = DataList::create($this->record->class)->byID($this->record->ID);
```

thanks!
Steve

**Comment by jebates** [comment]
Hi, thanks for pointing that out.

The "Versioning..." button you refer to is provided by Uncle Cheese's _Better Buttons_ module and the error happens when using that button as well as the "Cancel draft changes" button provided by Heyday's _Versioned Data Objects_ module.

We have made a new release (2.0.2) with a fix for this. Instead of using the slash, we included the use of the **DataList** class at the top of the **VersionedDataObjectDetailsForm**  class, for consistency, as shown below.

`use DataList;`

Thanks again.
